presto-docs/src/main/sphinx/security/kerberos-principals.fragment

```diff
@@ -0,0 +1,29 @@
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The fragments get included in both the server.rst and cli.rst. The content should be identical on both pages. I'll double check and make sure I don't have it verbatim rather than included somewhere.